<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindReplaceDialogClass</class>
 <widget class="QDialog" name="FindReplaceDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>170</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>170</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>170</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Find</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <property name="labelAlignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="formAlignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="labelFind">
         <property name="text">
          <string>Find</string>
         </property>
         <property name="buddy">
          <cstring>lineEditFind</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditFind"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelReplace">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Replace</string>
         </property>
         <property name="buddy">
          <cstring>lineEditReplace</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditReplace"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Options</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="checkBoxCaseSensitive">
          <property name="text">
           <string>Case sensitive</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxExactMatch">
          <property name="text">
           <string>Exact match</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxOnlyActiveTable">
          <property name="text">
           <string>Only active table</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonReplaceAll">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Replace all</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+Shift+R</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonReplaceNext">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Replace next</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+R</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonFindPrevious">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Find previous</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+Shift+G</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonFindNext">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Find next</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+G</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEditFind</tabstop>
  <tabstop>lineEditReplace</tabstop>
  <tabstop>buttonFindNext</tabstop>
  <tabstop>buttonFindPrevious</tabstop>
  <tabstop>buttonReplaceNext</tabstop>
  <tabstop>buttonReplaceAll</tabstop>
  <tabstop>checkBoxCaseSensitive</tabstop>
  <tabstop>checkBoxExactMatch</tabstop>
  <tabstop>checkBoxOnlyActiveTable</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
